### ISSUE TITLE
Just a quick question,

### ISSUE THREAD

**Issue by gjgfuj** [dnschneid/crouton#1356]
What's the difference between Xephyr and Xorg, I understand Xiwi is the chrome window, but what is the point of Xephyr? Thanks!

**Comment by dnschneid** [comment]
The xorg target doesn't work on some ARM platforms (exynos, mainly) due to driver issues.  The xephyr target abuses Chromium OS's X11 server to create a non-hardware-accelerated X11 server for crouton.  It's currently somewhat faster than xiwi, since xephyr bypasses the Chromium rendering system.  It's less flexible than xiwi since it can only be run full-screen, and I still have no idea how it behaves with multiple monitors.

X11 is [gradually being removed](http://www.chromium.org/developers/design-documents/ozone) from Chromium OS, so the xephyr target is not long for this world.  Ozone (a.k.a. Freon) also may bring about compatibility for unaccelerated-but-standalone xorg to systems that don't support it at all right now.

Basically, if you need hardware acceleration, use xorg.  If you don't need hardware acceleration and want something super-convenient, use xiwi.  If you want hardware acceleration but xorg doesn't work on your system, use xephyr, which is about as fast as it'll get for you.  There's no real reason to explicitly choose xephyr (other than switching away from xiwi), since if you don't explicitly specify an X server, it'll choose the fastest one your platform supports.

In the upcoming Freon (a.k.a. Ozone) world, expect xiwi to be the X11 method of convenience, and xorg to be the display method of better performance on any system (not necessarily hardware accelerated on all platforms, though).
